server.js

```diff
@@ -6,12 +6,12 @@ var electron = require('electron')
 console.log('STARTING SBOT')
 
 var createSbot = require('ssb-server')
-  .use(require('ssb-server/plugins/master'))
-  .use(require('ssb-server/plugins/logging'))
-  .use(require('ssb-server/plugins/unix-socket'))
-  .use(require('ssb-server/plugins/no-auth'))
-  .use(require('ssb-server/plugins/onion'))
-  .use(require('ssb-server/plugins/local'))
+  .use(require('ssb-master'))
+  .use(require('ssb-logging'))
+  .use(require('ssb-unix-socket'))
+  .use(require('ssb-no-auth'))
+  .use(require('ssb-onion'))
+  .use(require('ssb-local'))
```

Comment:
Oh, I was thinking that ssb-server *as a distro* should be including these by default, maybe I didn't understand correctly?